{"name": "Проверка ошибки при получении заказов пользователя без авторизации", "status": "passed", "steps": [{"name": "Получаем заказы конкретного пользователя", "status": "passed", "parameters": [{"name": "access_token", "value": "None"}], "start": 1713482897929, "stop": 1713482898256}], "start": 1713482897929, "stop": 1713482898258, "uuid": "93cd4bab-d1f4-4118-a6c4-84016b3753c7", "historyId": "c1954f6f4eb6058369536e2022675de9", "testCaseId": "c1954f6f4eb6058369536e2022675de9", "fullName": "tests.test_get_user_ordesr.TestGetUserOrders#test_get_user_orders_without_authorization", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_ordesr"}, {"name": "subSuite", "value": "TestGetUserOrders"}, {"name": "host", "value": "WIN-VD1QDI3F4U5"}, {"name": "thread", "value": "14400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_ordesr"}]}